src/test/java/com/rapid7/jul/LogentriesHandlerTest.java

```diff
@@ -0,0 +1,66 @@
+package com.rapid7.jul;
+
+
+import org.junit.After;
+import org.junit.Test;
+
+import java.util.logging.LogManager;
+import java.util.logging.Logger;
+
+import static junit.framework.TestCase.assertEquals;
+
+public class LogentriesHandlerTest {
+
+    @After
+    public void resetLogger() {
+        LogManager.getLogManager().reset();
+    }
+
+    @Test
+    public void singleLogger() throws Exception {
+        final String message = "single logger test";
+        final String token = "token-logentries-handler";
+        SocketChannelReceiver receiver = null;
+        try {
```

Comment:
Would be nice of `SocketChannelReceiver` implements `AutoCloseable` :)
```suggestion
        try (final SocketChannelReceiver receiver = SocketChannelReceiver.createAndStartReceiver(10000);) {
```

Reply:
You are right but the jdk version is set to Java 6.

Reply:
I think it's more important to keep Java6 compatibility here in that case.